Week3/assignment/transactions-create-tables.js

```diff
@@ -0,0 +1,50 @@
+//C:\Users\knowl\Documents\hyf\databases\Week3\assignment\transactions-create-tables.js
+import {createNewConnection,useDatabase} from "./connection_query.js";
+
+// Schema for the tables
+const accountTable = {
+    account_number: 'INT PRIMARY KEY',
+    balance: 'DECIMAL(10, 2) NOT NULL'
+};
+
+const accountChangesTable = {
+    change_number: 'INT AUTO_INCREMENT PRIMARY KEY',
+    account_number: 'INT NOT NULL',
```

Comment:
I would rename this to `from_account_number` and add a column named `to_account_number`